acceptance-tests/src/test/java/uk/gov/di/test/pages/SupportPage.java

```diff
@@ -58,4 +63,16 @@ public void switchToTabByIndex(Integer idx) {
         ArrayList<String> tabs = new ArrayList<String>(driver.getWindowHandles());
         driver.switchTo().window(tabs.get(idx));
     }
+
```

Comment:
Why is all of this in the Support step def? Think these would fit better in Login.java